int32 Car_Enable
int32 Car_Remote_Enable
int32 BMS_SOC
int32 BMS_Voltage
float32 BMS_Current
int32[4] WheelMotor_Enable
float32[4] WheelMotor_Spd
int32[2] TurnMotor_Enable
float32[2] TurnMotor_Angle